docs/apis/http/webdav/_index.md

```diff
@@ -0,0 +1,533 @@
+            <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+        <d:propstat>
+            <d:prop>
+                <oc:checksums></oc:checksums>
+                <oc:share-types></oc:share-types>
+                <d:getcontentlength></d:getcontentlength>
+                <d:getcontenttype></d:getcontenttype>
+            </d:prop>
+            <d:status>HTTP/1.1 404 Not Found</d:status>
+        </d:propstat>
+    </d:response>
+</d:multistatus>
+```
+{{< /tab >}}
+{{< tab "400 - Bad Request" >}}
+
+#### Body
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<d:error xmlns:d="DAV" xmlns:s="http://sabredav.org/ns">
+    <s:exception>Sabre\DAV\Exception\BadRequest</s:exception>
+    <s:message>Invalid Depth header value: 3</s:message>
+</d:error>
+```
+
+This can occur if the request is malformed e.g. due to an invalid xml request body or an invalid depth header value.
+{{< /tab >}}
+{{< tab "404 - Not Found" >}}
+
+#### Body
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<d:error xmlns:d="DAV" xmlns:s="http://sabredav.org/ns">
+    <s:exception>Sabre\DAV\Exception\NotFound</s:exception>
+    <s:message>Resource not found</s:message>
+</d:error>
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+### Request Body
+
+The `PROPFIND` Request can include an XML request body containing a list of namespaced property names.
+
+### Namespaces
+
+When building the body of your DAV request, you will request properties that are available under a specific namespace URI. It is usual to declare prefixes for those namespace in the `d:propfind` element of the body.
+
+Available namespaces:
+
+| URI                                       | Prefix |
+|-------------------------------------------|--------|
+| DAV:                                      | d      |
+| http://sabredav.org/ns                    | s      |
+| http://owncloud.org/ns                    | oc     |
+| http://open-collaboration-services.org/ns | ocs    |
+| http://open-cloud-mesh.org/ns             | ocm    |
+
+### Request Example with declared namespaces
+
+```xml
+<?xml version="1.0"?>
+    <d:propfind xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns">
+    </d:propfind>
+```
+
+### Supported WebDAV Properties
+
+| Property                    | Desription                                                                 | Example                                                                                                                                          |
+|-----------------------------|----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| `<d:creationdate />`        | The creation date of the resource.                                         | `1970-01-01T00:00:00+00:00`                                                                                                                      |
```

Comment:
asking for the creationdate property always returns 404 for me
```
  <d:response>
    <d:href>/dav/files/admin/Foto.png</d:href>
    <d:propstat>
      <d:prop>
        <d:getlastmodified>Mon, 25 Sep 2023 14:07:31 GMT</d:getlastmodified>
      </d:prop>
      <d:status>HTTP/1.1 200 OK</d:status>
    </d:propstat>
    <d:propstat>
      <d:prop>
        <d:creationdate/>
      </d:prop>
      <d:status>HTTP/1.1 404 Not Found</d:status>
    </d:propstat>
  </d:response>
```

Reply:
should i remove it?

Reply:
if it does not work in ocis, maybe better to remove it